minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java

```diff
@@ -52,6 +52,10 @@ public AbstractContainer(MesosCluster cluster, String uuid, String containerId,
         this.config = config;
     }
 
+    private int getSystemPropertyLength(String property) {
```

Comment:
class Utilities {
  private static String magicWord = "magic";

  private String getMagicWord() { // Noncompliant
    return magicWord;
  }

  private void setMagicWord(String value) { // Noncompliant
    magicWord = value;
  }

}


This my guide code? And edit this code and tiling you how many code I change.